### ISSUE TITLE
Ubuntu PPA new version

### ISSUE THREAD

**Issue by brcha** [zealdocs/zeal-packaging#27]
Hello,

Thank you for building an excellent documentation app. I would like to ask you if it would be possible to update the app in Ubuntu's PPA to the current version as well as add support for zesty (17.04) if that is not too much trouble.

Thank you in advance :)
brcha

**Comment by Xuanwo** [comment]
Same here, if there is anything I can make a help, please let me know.

**Comment by JoseBarakat** [comment]
Same here...

```
jose@jose-Lenovo-G400s:~$ uname -a
Linux jose-Lenovo-G400s 4.10.0-35-generic #39-Ubuntu SMP Wed Sep 13 07:46:59 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```
```
jose@jose-Lenovo-G400s:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 17.04
Release:	17.04
Codename:	zesty
```
```
jose@jose-Lenovo-G400s:~$ zeal -v
Zeal 0.3.1
```

**Comment by trollixx** [comment]
Fixed by #29.
